dotcom-rendering/src/components/StarRatingBlockComponent.tsx

```diff
@@ -9,12 +10,18 @@ type Props = {
 };
 
 const starsWrapper = css`
-	background-color: ${brandAltBackground.primary};
+	background-color: ${palette('--star-rating-background')};
 	display: inline-block;
 `;
 
 export const StarRatingBlockComponent = ({ rating, size }: Props) => (
 	<div css={starsWrapper}>
-		<StarRating rating={rating} size={size} />
+		<StarRating
+			rating={rating}
+			size={size}
+			// This colour is hardcoded, as background changes to reflect dark/light mode
+			// and we want the stars to always be the same colour
+			fill={sourcePalette.neutral[7]}
```

Comment:
Not sure I understand what that looks like? Stars are always very dark? Why is this not `palette('--star-rating-fill')`?

Reply:
While that makes sense, I wonder what the benefit of `--star-rating-fill`, then.

I would imagine that star rating should have only two fills:
1. `currentColor`, based on the colour of surrounding text
2. `var(--star-rating-fill)`, based on a value that contrasts enough with `var(--star-rating-background)`

Using [`currentColor` is both more semantic and flexible](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#currentcolor_keyword) than a CSS custom property for the standalone `Star` component.

Reply:
None!